interface/src/app/gene/searchFew/autocomplete-search-panel.tpl.html

```diff
@@ -1,16 +1,12 @@
 <div class="panel panel-default">
   <div class="panel-heading">
-    <h3>
-      Search for Genes
-    </h3>
+    <b>Type in gene name to search using autocomplete</b>
```

Comment:
I think this succinct instruction is just right. Even using the term "autocomplete" (which I complained about on the button above) is helpful in this context, as it gives an idea of what is about to happen *right now* when the feature is used.